.changeset/tidy-lemons-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/typography': minor
```

Comment:
shoot i merged this - but this needs to be major. can you update in your select PR @TheSonOfThomp